README.md

```diff
@@ -70,3 +70,7 @@ script/local_e2e
 ```
 
 Note: This requires `ap-tools` to be installed (<https://github.com/ministryofjustice/hmpps-approved-premises-tools>)
+
+## Release process
+
+Our release process aligns with the other CAS teams and as such [lives in Confluence](https://dsdmoj.atlassian.net/wiki/spaces/AP/pages/4247847062/Release+process)
```

Comment:
Perhaps the README should also link to [.github/PULL_REQUEST_TEMPLATE.md](https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/pull/68/commits/ce41a72cb84b8f6dedbf71f06e82efcf49f13c33#diff-18813c86948efc57e661623d7ba48ff94325c9b5421ec9177f724922dd553a35) so that more detail is readily discoverable from within the repo itself?

There's always a risk when linking to other resources link Confluence that the viewer won't have access or the wiki page is moved/removed...